Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -373,6 +373,8 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
             )
             if response.status_code // 100 != 2:
                 sys.stderr.write("Failed to modify '{}'\n".format(issue))
+                sys.stderr.write("Request was URL {} with content {}\n".format(response.url, update_dict))
```

Comment:
This is going to be a lot of spew. Thoughts on sending this to `log.info` so it's tied to `-v`?

Reply:
This will only run if the request fails. In that case, the user probably wants to know what went wrong with it and won't mind the spew. Which most likely won't be that much. I just saw one line of JSON saying "You need admin permissions in this repo" or something of the like.

I don't think binding it to log.info() it's a good idea because then the user wouldn't know what went wrong until the second try of whatever command they were running, which may end up going through different codepaths.